cpp/src/arrow/filesystem/azurefs.cc

```diff
@@ -611,6 +611,110 @@ class AzureFileSystem::Impl {
     RETURN_NOT_OK(ptr->Init());
     return ptr;
   }
+
+  Status CreateDir(const AzurePath& path) {
+    if (path.container.empty()) {
+      return Status::Invalid("Cannot create an empty container");
+    }
+
+    if (path.path_to_file.empty()) {
+      auto container_client =
+          blob_service_client_->GetBlobContainerClient(path.container);
+      try {
+        auto response = container_client.Create();
+        if (response.Value.Created) {
+          return Status::OK();
+        } else {
+          const auto& body = response.RawResponse->GetBody();
+          std::string_view body_text(reinterpret_cast<const char*>(body.data()),
+                                     body.size());
```

Comment:
Is it safe to put this in a string? We are sure this is plain text?

Reply:
I hope so. I expect that the body includes a detailed error message and it's helpful for debug.
But it's not an Azure spec.
Should we avoid this for safety?
Or can we keep this as-is with a concern comment?

Reply:
It's fine to keep as is.

Reply:
OK. I'll add a comment.